R/pkg/R/SQLContext.R

```diff
@@ -446,14 +446,15 @@ dropTempTable <- function(sqlCtx, tableName) {
 #' @param source the name of external data source
 #' @return DataFrame
 #' @export
+#' @aliases loadDF
 #' @examples
 #'\dontrun{
 #' sc <- sparkR.init()
 #' sqlCtx <- sparkRSQL.init(sc)
-#' df <- load(sqlCtx, "path/to/file.json", source = "json")
+#' df <- read.df(sqlCtx, "path/to/file.json", source = "json")
 #' }
 
-loadDF <- function(sqlCtx, path = NULL, source = NULL, ...) {
+read.df <- loadDF <- function(sqlCtx, path = NULL, source = NULL, ...) {
```

Comment:
I didn't know this was a valid way to declare alias functions in R. Hmm - But it seems to go against the style in the rest of the file. Could we just declare a new function `read.df` similar to other aliases ?